dockerfile/anaconda-live-iso-creator/Dockerfile.j2

```diff
@@ -54,11 +58,13 @@ RUN set -ex; \
 # required for ksflatten
   pykickstart; \
   mkdir /fedora-kickstarts; \
-{% if distro_release == "rawhide" %}
-  git clone -b main https://pagure.io/fedora-kickstarts.git /fedora-kickstarts; \
-{% else %}
-  git clone -b f{$ distro_release $} https://pagure.io/fedora-kickstarts.git /fedora-kickstarts; \
-{% endif %}
+  DISTRO_RELEASE="${CI_TAG#fedora-}"; \
+  [ "$DISTRO_RELEASE" = "$CI_TAG" ] && DISTRO_RELEASE="$CI_TAG"; \
```

Comment:
This line is not needed, because this code only runs with CI_TAG fedora-*

RHEL live iso builds are not supported.